src/Symfony/Component/Validator/Constraints/FileValidator.php

```diff
@@ -138,10 +138,12 @@ public function validate($value, Constraint $constraint)
         }
 
         $sizeInBytes = filesize($path);
+        $basename = ($value instanceof UploadedFile) ? $value->getClientOriginalName() : basename($path);
```

Comment:
brackets should be remove

Reply:
Removed as requested - I use brackets for readability purposes, no side effects